PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -52,6 +62,30 @@ public void setListener(PayPalNativeCheckoutListener listener) {
         this.listener = listener;
     }
 
+    /**
+     * @param activity      Android FragmentActivity
+     * @param payPalRequest a {@link PayPalNativeRequest} used to customize the request.
+     * @deprecated see {@link PayPalNativeCheckoutClient#launchNativeCheckout(FragmentActivity, PayPalNativeRequest)}
+     * <p>
+     * Tokenize a PayPal account for vault or checkout.
+     * <p>
+     * This method must be invoked on a {@link PayPalNativeCheckoutClient (Fragment, BraintreeClient)} or
+     * {@link PayPalNativeCheckoutClient (FragmentActivity, BraintreeClient)} in order to receive results.
+     */
+    @Deprecated
+    public void tokenizePayPalAccount(@NonNull final FragmentActivity activity, @NonNull final PayPalNativeRequest payPalRequest) throws Exception {
+        // NEXT_MAJOR_VERSION: remove tokenizePayPalAccount method and refactor tests to center
+        // around launchNativeCheckout in the future. Keeping the tests as they are for now allows
+        // us to maintain test coverage across both the tokenizePayPalAccount and launchNativeCheckout methods
+        boolean isCheckoutRequest = payPalRequest instanceof PayPalNativeCheckoutRequest;
+        boolean isVaultRequest = payPalRequest instanceof PayPalNativeCheckoutVaultRequest;
+        if (isCheckoutRequest || isVaultRequest) {
+            launchNativeCheckout(activity, payPalRequest);
+        } else {
+            throw new Exception("Unsupported request type");
```

Comment:
Should we make this message similar to the one we are using below to make it more explicit to a merchant what they are doing wrong? So something like "Unsupported request type. Please use either a PayPalNativeCheckoutRequest or a PayPalNativeCheckoutVaultRequest."?

Reply:
We could yeah. I wasn't as concerned with this one since it's deprecated, but we could add the same message.